.github/CODEOWNERS

```
@@ -0,0 +1,15 @@
+# default (to be overridden by any matches in later lines below)
+*                        @rapidsai/jupyterlab-nvdashboard-write
+
+# CI code owners
+.flake8                  @rapidsai/ci-codeowners
```

I would've put this under packaging as it is a linter config that most closely aligns with `pyproject.toml` -- but it's fine either way.

---

🤷🏻 the line between "packaging" and "CI" is a thin one

---

If we list both teams, GitHub treats it as `or`

```suggestion
.flake8                  @rapidsai/ci-codeowners @rapidsai/packaging-codeowners
```

---

ok that's fine too